test/expected/default-option-dependencies.json

```
@@ -14,6 +18,8 @@
     "src/modules/desktop/**/*.html",
     "src/modules/tablet/**/*.html",
     "src/modules/tv/**/*.html",
+    "lib/watch/apple/**/*.html",
```

hey @urgoringo this is a great question. This gunt task will only collectDependencies that have a 'Module.js' file in its directory. In the case that we try to collect `tw.common.errors` and `lib/common/errors` doesn't have a `Module.js` file in it, then the dependency would not be collected AND NO EXCEPTIONS WOULD BE RAISED. 

So the real question would be if we should throw an error for these cases. I think we should. What do you think?